decoder/vaapidecoder_vp8.cpp

```diff
@@ -564,4 +567,28 @@ YamiStatus VaapiDecoderVP8::decode(VideoDecodeBuffer* buffer)
     return status;
 }
 
+bool VaapiDecoderVP8::isSVCTFrame()
+{
+    switch (m_configBuffer.temporalLayer) {
+    case 0:
+        return true;
+    case 1:
+        if ((m_frameHdr.key_frame == Vp8FrameHeader::KEYFRAME) ||
+            m_frameHdr.refresh_last)
+            return true;
+        break;
+    case 2:
+        if ((m_frameHdr.key_frame == Vp8FrameHeader::KEYFRAME) ||
+            m_frameHdr.refresh_last || m_frameHdr.refresh_golden_frame)
+            return true;
+        break;
+    case 3:
+        return true;
+    default:
```

Comment:
Please return true for default, if you have 3 layers, user want 4 layers, you need decode all 3 for them